CMSgov/design-system
# [WNMGDS-2238] Healthcare privacy settings modal button css cascading incorrectly.

## Summary

- updates hover color for privacy settings button in healthcare footer to match other links
- updates cascade to only style the first child button in this case so it doesn't override button in privacy settings modal

## How to test

1. `yarn clean && yarn install && yarn build && yarn storybook`
2. Open [http://localhost:6006/?path=/story/healthcare-footer--basic-footer&globals=theme:healthcare](http://localhost:6006/?path=/story/healthcare-footer--basic-footer&globals=theme:healthcare)
3. Inspect privacy settings modal button, confirm it does not include reset styles from `.hc-c-footer__inline-item > button`

## Checklist
